tests/gemm/test_mxfp8_linear_fused_quant_a.py

```
@@ -0,0 +1,95 @@
+separate path, eagerly and under CUDA-graph capture.
+
+Bit identity holds when the separate path reduces split-K partials in FP32
+(B12X_DENSE_SPLITK_TURBO=0, the production setting); the bf16 atomic
+split-K accumulation is itself order-dependent, so the module constant is
+pinned before the GEMM module is imported.
+"""
+import os
+
+os.environ.setdefault("B12X_DENSE_SPLITK_TURBO", "0")
+
+from __future__ import annotations
```

Ruff (0.16.3)

[error] `from __future__` imports must occur at the beginning of the file

(F404)